src/types.ts

```diff
@@ -75,11 +75,16 @@ export interface UADataValues {
     architecture: string;
     model: string;
     uaFullVersion: string;
+    fullVersionList: NavigatorUABrandVersion[];
 }
 export interface NavigatorUAData {
     brands?: NavigatorUABrandVersion[];
     uaList?: NavigatorUABrandVersion[];
     mobile: boolean;
     platform: string;
-    getHighEntropyValues<T extends keyof UADataValues>(hints: T[]): Promise<{ [key in T]: UADataValues[T] }>;
+    getHighEntropyValues<T extends keyof UADataValues>(hints: T[]): Promise<{
```

Comment:
UADataValues 대신 HighEntropyValues를 쓰지 않아도 될 것 같습니다.
UADataValues와 동일한 타입같습니다.

---

I don't think you should use HighEntropyValues instead of UADataValues.
Same type as UADataValues.

Reply:
HighEntropyValues를 사용하지 않으면, getAccurateAgent의 getClientHintsAgent를 사용하는 부분에서 타입 에러가 발생해서 추가했습니다.
(https://github.com/naver/egjs-agent/blob/071d9cbf18c473bd80aaf0433e73e7b0726d941b/src/agent.ts#L35)

```
(parameter) info: {
    architecture: string | NavigatorUABrandVersion[];
    platform: string | NavigatorUABrandVersion[];
    platformVersion: string | NavigatorUABrandVersion[];
    model: string | NavigatorUABrandVersion[];
    uaFullVersion: string | NavigatorUABrandVersion[];
    fullVersionList: string | NavigatorUABrandVersion[];
}
'{ platform: string | NavigatorUABrandVersion[]; platformVersion: string | NavigatorUABrandVersion[]; architecture: string | NavigatorUABrandVersion[]; model: string | NavigatorUABrandVersion[]; uaFullVersion: string | NavigatorUABrandVersion[]; fullVersionList: string | NavigatorUABrandVersion[]; }' 형식의 인수는 'UADataValues' 형식의 매개 변수에 할당될 수 없습니다.
  'platform' 속성의 형식이 호환되지 않습니다.
```